### ISSUE TITLE
License is incompatible

### ISSUE THREAD

**Issue by stweil** [OCR-D/ocrd_olena#20]
Please add Apache 2.0 or MIT license.

**Comment by stweil** [comment]
Some other [OCR-D projects](https://github.com/OCR-D) also don't show a license. Maybe those can be updated, too.

**Comment by kba** [comment]
We should indeed add explicit licenses throughout. All OCR-D projects are `Apache 2.0` licensed.

**Comment by stweil** [comment]
@kba, I'm afraid that we have to rethink the license issue because olena uses GPL2.

**Comment by kba** [comment]
Indeed, it's unfortunate. We could ask @glazzare or @roland-levillain since they are/were the main contributors but there is a long list of other contributors which makes this unfeasible even if we agreed to relicense.

So, yeah, ocrd_olena will have to be GPL too.

**Comment by stweil** [comment]
See pull request #90.
